src/implementations/twig/components/ecl-component-button/button.test.js

```diff
@@ -1,14 +1,14 @@
 import { merge, renderTwigFileAsNode } from '@ecl/test-utils';
 
 // Import data for tests
-import dataPrimary from '@ecl/ec-specs-button/demo/data--primary';
-import dataSecondary from '@ecl/ec-specs-button/demo/data--secondary';
-import dataCall from '@ecl/ec-specs-button/demo/data--call';
-import dataGhost from '@ecl/ec-specs-button/demo/data--ghost';
-import dataSearch from '@ecl/ec-specs-button/demo/data--search';
+import dataPrimary from '@ecl/ecl-specs-button/demo/data--primary';
```

Comment:
The number of stories growing, please organize them like here https://github.com/ec-europa/europa-component-library/blob/v2-dev/src/systems/ec/implementations/react/components/page-banner/stories/Index.jsx

Reply:
Mmhh..this surprises me, basically this would mean creating a file per variant, to export a single story, right? We said we would try to keep the duplication of code under control, this would introduce lots of it, no? Many more files to handle, at least.
Not sure, then, if we want to move to the current approach in ECL v2 it's worth maybe doing it in a separate ticket for both the "demo" components we have, but i'm not sure this would be an advantage, at least i think it would be worth discussing it before making that move.

Reply:
What I meant was having 1 story file https://github.com/ec-europa/europa-component-library/blob/98f4647b41f805ce05a0bb65518ccca69ca669f1/src/implementations/twig/components/ecl-component-button/button.story.js of 127 lines of code and repetition being split into several smaller files. Duplication topic is WTF here. Rather it's a usual approach of having smaller files to work on, that's literally all i'm asking for.

Reply:
Skip it if you prefer btw, all the same for me

Reply:
Yeah, but that's what i intended, lots of files to deal with, i'm not really discussing a general pattern in software development, neither expressing my preference about it, just considering the opportunity of making such a decision in the context of this pr and only for the button. I won't skip it, anyway.